content/jobs/open/product-community-lead-2022.md

```diff
@@ -0,0 +1,84 @@
+* **Provide one-on-one or group guidance to our user communities** (particularly leaders in those communities) with the goal of transferring knowledge that they can then share within their peers.
+
+**Foster deeper external relationships and engagements** with key communities of practice and partners that are important for 2i2c’s strategy, such as the [Pangeo Project](pangeo.io) (which is a key stakeholder for this role).
+
+* **Engage with potential communities and stakeholders**, with a focus on demonstrating the use-cases we hope to support and leading to new service contracts with these communities.
+* **Communicate externally to demonstrate impact.** For example, by participating in meetings, conferences, and other events where we have an opportunity to highlight the work that 2i2c does.
+* **Interact extensively with open source communities** that underlie our infrastructure, potentially serving in leadership roles and doing significant work to support these communities. There will be a particular focus on the Jupyter and PyData ecosystems.
+
+### Guide product design and implementation
+
+This role should understand the perspectives of our users and the use-cases we wish to enable, and bring this perspective to help guide the evolution of our services and infrastructure. Below are a few example responsibilities:
+
+**Represent user communities and communicate their needs** to the engineering team, leadership team, and other stakeholders. For example:
+
+* **Discuss and shape ideas for development** in collaboration with the engineering and sustainability teams within 2i2c. This may entail writing design documents and participating in GitHub issues and Pull Requests.
+* **Discuss and shape how to evolve our services.** Beyond focused development, this role should also help shape our strategic plan for evolving our service offerings, or designing services for new use-cases.
+
+**Coordinate with and guide our team in implementation** as we carry out development and service design. For example:
+
+* **Participate in team planning meetings and practices.** For example, prioritization and planning in GitHub Issues, or in synchronous team meetings.
+* **Collaborate with engineers throughout the implementation of enhancements to our infrastructure**. For example, providing ongoing check-ins, guidance, and feedback via GitHub issues.
+
+### Strategic leadership
+
+This is the first position of its kind within 2i2c, and it will have freedom to shape its strategy and role with the organization. It should oversee the evolution and growth of this _kind _of work within 2i2c, and develop organizational strategy for how to maximize our impact. Below are a few ways that this might occur (both independently and in partnership with others at 2i2c):
```

Comment:
```suggestion
This is the first position of its kind within 2i2c, and it will have freedom to shape its strategy and role with the organization. It should oversee the evolution and growth of this _kind_ of work within 2i2c, and develop organizational strategy for how to maximize our impact. Below are a few ways that this might occur (both independently and in partnership with others at 2i2c):
```